examples/languages/react/src/App.tsx

```diff
@@ -1,84 +1,101 @@
+        console.log(`New connection linked with ID: ${connection.connectionId}`);
+    }
+    const onClose = () => setModalOpen(false);
+    const onFinish = () => {
+        onClose();
+        setIsFinished(true);
+    }
+    const onError = (error: ErrorCallbackArgs) => {
+        // this error should be logged in your error tracking service
+        console.error(`Codat Link SDK error`, error);
+        if (!error.userRecoverable) {
+            onClose();
+        }
+    }
 
-  return (
-    <div className="App">
-      <header>
-        <h1>
-          <a
-            target="_blank"
-            rel="noreferrer"
-            href="https://github.com/codatio/sdk-link/tree/main/examples/react/README.md"
-          >
-            Link SDK X Create React App
-          </a>
-        </h1>
-        <p>
-          An example of embedding Link SDK within a brand new create-react-app
-          project
-        </p>
-      </header>
+    return (
+        <div className="App">
+            <header>
+                <h1>
+                    <a
+                        target="_blank"
+                        rel="noreferrer"
+                        href="https://github.com/codatio/sdk-link/tree/main/examples/react/README.md"
+                    >
+                        Link SDK X Create React App
+                    </a>
+                </h1>
+                <p>
+                    An example of embedding Link SDK within a brand new create-react-app
+                    project
+                </p>
+            </header>
 
-      <div>
-        <ol>
-          <li>
-            <a
-              href="https://github.com/codatio/sdk-link/tree/main#create-a-new-company"
-              target="_blank"
-              rel="noreferrer"
-            >
-              Create a company
-            </a>
-          </li>
-          <li>
-            <a>Paste the company ID below</a>
-          </li>
-        </ol>
-        <div className="input-wrapper">
-          <input
-            value={companyId}
-            placeholder="Provide a valid company ID"
-            onChange={(e) => {
-              setCompanyId(e.target.value);
-            }}
-          />
-          <button onClick={() => setModalOpen(!modalOpen)}>
-            {modalOpen ? "Exit" : "Start authing"}
-          </button>
+            <div>
+                <ol>
+                    <li>
+                        <a
+                            href="https://github.com/codatio/sdk-link/tree/main#create-a-new-company"
+                            target="_blank"
+                            rel="noreferrer"
+                        >
+                            Create a company
+                        </a>
+                    </li>
+                    <li>
+                        <a>Paste the company ID below</a>
+                    </li>
+                </ol>
+                <div className="input-wrapper">
+                  {(isFinished? (<>
```

Comment:
@Rachel-Codat Is this the part you're asking about in https://teamcodat.slack.com/archives/C0560S3N0AW/p1727090675234869?thread_ts=1726842129.619269&cid=C0560S3N0AW?